modules/rostests/winetests/riched20/txtsrv.c

```diff
@@ -48,7 +48,7 @@ static PCreateTextServices pCreateTextServices;
 
 /* Use a special table for x86 machines to convert the thiscall
  * calling convention.  This isn't needed on other platforms. */
-#if defined(__i386__) && !defined(__MINGW32__)
+#if  defined(__i386__) && !defined(__MINGW32__) && (!defined(_MSC_VER) || !defined(__clang__))
```

Comment:
```suggestion
#if defined(__i386__) && !defined(__MINGW32__) && (!defined(_MSC_VER) || !defined(__clang__))
```

Reply:
This extra space was present in Wine-6.10, Changed. Thanks.

Reply:
Please don't ever change formatting in 3rd party code. It's not only useless, it's making syncs harder.

Reply:
I accidentally removed the extra space somehow. It should be fixed now. Thanks.